docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,127 +4,178 @@
+. *Configure {kib} to access the certificate and private key.*
++
+You need to specify the information required to access your {kib} server's certificate and corresponding private key.
+
+.. If your certificate and private key are in PEM format:
 +
 --
+Specify your certificate and private key in `kibana.yml`:
+
 [source,yaml]
 --------------------------------------------------------------------------------
-server.ssl.enabled: true
+server.ssl.certificate: "/path/to/kibana-server.crt"
+server.ssl.key: "/path/to/kibana-server.key"
+--------------------------------------------------------------------------------
+
+If your private key is encrypted, add the decryption password to your <<secure-settings,{kib} keystore>>:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+bin/kibana-keystore add server.ssl.keyPassphrase
 --------------------------------------------------------------------------------
 --
 
-. Specify your server certificate and private key in `kibana.yml`:
+.. Otherwise, if your certificate and private key are in a PKCS#12 file:
 +
 --
-If your certificate and private key are in a PKCS #12 keystore, specify it like so:
+Specify your PKCS#12 file in `kibana.yml`:
 
 [source,yaml]
 --------------------------------------------------------------------------------
-server.ssl.keystore.path: "/path/to/your/keystore.p12"
-server.ssl.keystore.password: "optional decryption password"
+server.ssl.keystore.path: "/path/to/kibana-server.p12"
 --------------------------------------------------------------------------------
 
-Otherwise, if your certificate/key are in PEM format, specify them like so:
+If your PKCS#12 file is encrypted, add the decryption password to your <<secure-settings,{kib} keystore>>:
 
 [source,yaml]
 --------------------------------------------------------------------------------
-server.ssl.certificate: "/path/to/your/server.crt"
-server.ssl.key: "/path/to/your/server.key"
-server.ssl.keyPassphrase: "optional decryption password"
+bin/kibana-keystore add server.ssl.keystore.password
 --------------------------------------------------------------------------------
 
-After making these changes, you must always access {kib} via HTTPS. For example, 
-https://localhost:5601.
+TIP: If your PKCS#12 file isn't protected with a password, depending on how it was generated, you may need to set
+`server.ssl.keystore.password` to an empty string.
+--
+
++
+For more information about settings for certificates and keys, see <<settings,{kib} configuration settings>>.
 
-For more information, see <<settings,{kib} configuration settings>>.
+. *Configure {kib} to enable TLS for inbound connections.*
++
 --
+Specify that TLS/SSL is used in `kibana.yml`:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+server.ssl.enabled: true
+--------------------------------------------------------------------------------
+--
```

Comment:
I moved this section down, as it should be the last change to the `kibana.yml` file.